app/models/project.rb

```diff
@@ -9,7 +9,7 @@ class Project < ActiveRecord::Base
   has_many :orders, through: :loans
   belongs_to :tenant
 
-  validates :description, length: { in: 100..255, allow_nil: false },
+  validates :description, length: { in: 50..255, allow_nil: false },
```

Comment:
I think 100 characters is really the bare minimum for a description for a project. If you pulled from master, the seed files and all the non-skipped tests are set to meet this criterion.